src/components/member-role-update/index.js

```diff
@@ -53,21 +57,41 @@ const MemberRoleUpdate = () => {
   const renderPromoteButton = () => {
     return (
       <>
-        <button
-          className={classNames.moveToMember}
-          type="button"
-          onClick={() => promoteToMember(selectedMember)}
-        >
-          Promote to Member
-        </button>
+        {member ? (
+          <button
+            className={classNames.moveToMember}
+            type="button"
+            onClick={() => promoteToMember(selectedMember)}
+          >
+            Demote Member
+          </button>
+        ) : (
+          <button
+            className={classNames.moveToMember}
+            type="button"
+            onClick={() => promoteToMember(selectedMember)}
+          >
+            Promote to Member
+          </button>
+        )}
+        {archived ? (
+          <button
+            className={classNames.moveToMember}
+            type="button"
+            onClick={() => archiveTheMember(selectedMember)}
+          >
+            Unarchive Member
+          </button>
+        ) : (
+          <button
+            className={classNames.moveToMember}
+            type="button"
+            onClick={() => archiveTheMember(selectedMember)}
+          >
+            Archive Member
+          </button>
+        )}
```

Comment:
can we put these in a contant, and then add it here?

Reply:
fixed